demo/src/components/CodeModal/CodeModal.js

```diff
@@ -0,0 +1,53 @@
+import React, {useState, useEffect} from 'react';
+import {Button, Modal, Tabs, Tab} from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import './CodeModal.css';
+import Prism from 'prismjs';
+import 'prismjs/themes/prism.css';
+
+CodeModal.propTypes = {
+  popupId: PropTypes.string,
+  ourCode: PropTypes.string,
+  gtagCode: PropTypes.string,
+};
+
+/**
+ * A popup to display the google analytics code that fires on
+ * a button click.
+ * @param {string} popupId The ID of this popup.
+ * @param {string} ourCode The code for this open source library.
+ * @param {string} gtagCode The code for gtag.
+ * @return {!JSX} The component.
+ */
+export function CodeModal({popupId, ourCode, gtagCode}) {
```

Comment:
nit: I'd change this to something like "measurementLibraryCode"

Reply:
I was planning to go back and change it once we've decided on an official name, I'll bring this up  at the standup today

Reply:
Fixed